readme/data_compliance_db_tables.drawio

```diff
@@ -0,0 +1 @@
w+CYIk+3eURuts/LA0C3ivK1pQRm5tFsl0TXyoGrqkbiXm5bzTjXB5BIAT5xk+e7yZC9E2pLGFcElmkZtFjqvOcJaPgY7APSuSRJzEPgijFWTcdLki3s7QUKTo1xWXc/JncXNnuHjKnFaLnJPQ0rBks5KRm3WTAvBeoj4oYU5+/WRUa8xi84kql+oNdJqA9J3a7qyTBDIq3zLg4h22HOXfHMixpFjQUjXsYQq2JejM/tzuDB76qzHAMTkOzVzDsnCLE2N10aDCzQb25ipSMvLxQHgRGTlXpZZSm5qesVtTvvGK5GiUQJ1n5HzxHDCMCpURnp6q046L3UrzuC8jXm2q4w5WaGlcKJTdbrgGQ3pCVPMtZsVkAVqsVvZk39Rd1mG2LLTu6sBygRZSruF2JD86bQR+N7Dhqz3YHAG57rIa0wMO+k/wPlfiag97imGuODuNcfUST4qnA3x/ml12F+1Ut5uz1gtGcd5BfR0287bjotW4Rp1faNT82Pta5f9VidQqiGc60mouHkZXN2LZiqN7zZDvOozum8kgkNixE/GYriONHNoa7W7XLEGvZAkSpKzyvX5QIMhvkSmwMA49HIZxzNzMQX66EOgqyZ0KsuCg+Zdy06uxbr7HDOrM5aE4qEQJWN26SGWEOhzNjWGLjiUTZFxO+arNowX4OkdDFoVZmP1TizUUjK4V5sjwutuPSjiLu13WoPtDzA2hJhUIK/dKt3RZBbmfDUfD0pgXqKYULuDBZ3qTHILgj9cj0ipG9t9xia5JbLekb5ISFJnshFWoFaoZzuPqSNsfZpBcANzDHqYF+cgds9lN0bB8tTtVDd0S0ycMprbrb3hGu/wSzjP+tXSoM6J+UCeHOOCdpV9jjE99QN97w12BuJVXp+nfgDQezpxO5gaIAE6QbpNt6UKGVhsk9QYJRwlE0KfrRael+r6uhXXTmjvj26Anbb4rsgoZYKZDp4tj94SWYvRUvemC02bHX6exRrg/BOvR4mkcxShw2irJ1Zn1QzFar+3+A2MyKfZdiFqoIFVbmVCHfIotFQl31rA0Q0Bst7xVIxzPkBLti+tt3JiDk1BTkY/Wpec6yL/CmOVuIadbwT7tSKF4zbZbNCVeZwUxeMOSbhZNwOum6pUEe4c6ZdsZ/EBWhYWShnKiXMkETts7RGmGCR6RlhZxk8x3dYX3G7iaAquagzhZNBJ7ThnLK2bvFZi522gLlgJxkpLaGjGMc5kJO2b/Wg3un3J1qCL4DtK9lYEPrzIRlTJDkGbQ2y1ZZQll6r0YqBLDvTGZYvtqr6kGnx0p5WRe+s1YhhecTAYGIb6Cec0nUIpc6zHnE6qGKPAvKVy3uLoEEDBaq7ptnJkLTcOzhBPQiT4YUN35BQsaHWjT5v2VO97lmPTRD/yrjQsjdBsV/vNQnWQzW6SMofwPbTKJuHMH+XwtKbaVr+InFnxnogCG972m013jeXMP11P+faapCS6mSYrO6Rmdp3UMOIvSOe3usOFnM5UrDTWR4OxlL7IUIaPaHe5oixBvC7yBIHeeE2V1xxCOludpRN54gNo2MOdRWnShsUcCOiieBNW0O5vzWt2IYKqNyls17FOpgeXc66H2AUMjnw4YYYkbPqqx8FAR61l4PqaRqxoUqmR6xfuwfUQIGr8QVny7QY4r5XaUvQB2F6I5z2I6SlPXLJWuzUTkvIRtqi1hdfwi0jeGBC3mDwjalDOakOvudyzugYdmFmGj/ftsEpP8eq6PaCcasieMORbg1WchI4s7WSclpwoSS7eO5lUWc4ahlDa2pxsGEmONe6k19HecKbqnHv6Lq0uVbJbOoNVC8KXZoPePsFO0R9PjZCvTbAjxD/0K7M8JPmDptiZ33F50t0+mbv925zzZ23S25bl9EBw4dKaLgnOv3iF0k/cKyMvsTeyVQfHIag+PHXUe1+BHSUfoiZOPqRVkGVJ/Ne/20PzY/bDd0nyAVjH/otM/lk3/gtWe+Hsiwn/17YC468tOMB+1GqvhyUH/71rWj5d7/WWNS3Ye13U8iVA8W4WtmAv9kfgzDeubEEp+h8GoQn24fO6/vysZeW/4bpybcUp0gfP1Nec+IHnXEH9A92+7kTDoI/y93PqztBWTRB/uDMzP/9skW9lF1jB34fZCKhZ/26Z/FmM/QQ2/qDHb0SPf7/cnkW9gh5fPUjmh+0VeHiv38khPu4af+4Z//jENxvKX5fEeGdO+ZUTPt6d33ja2g7XOL/n9EoaRO+av5v0/z4Hu/3x0b/ER+MMzDh8tKcP+dRPowz9z2vHAxA/zFP/0CMCvpTnaX5Cjuc33rb05n192HvL8fyDss8+DEN9JPUo+Y0pn79Z9h+WxUjk9nl48Zfa9LNyPr/hCUlPh9rdY9w/R9v9BzD3Him9hwPuXuPh18/ZfXqC1A8LDPD/uKO+B09/wM+3gp8X21BJ9pUEBfZzExTv5rTb+TvAHOxbj7r9HOp5PP7gDajn+x+R++9wzr89lwBHPpZW4iX2fjNgwdCPAT/xsyHKa/uv3zlEebme6IOwFv/Akze53xeLiu7Wo7ybpFhcnNsqmN/3XxL4Hoz9QQnfiBI+OWTujWtgqB+1GhP9DZdjrjjDuzsr7mZF/1jOr2v9FxYbvpcs/TueSXjqtvc/pQ9ga5q+58mEPw7oFzogCvn632N5dRHmy+OZvp8HYn9/D/RnX8C3O6Jfvj3gF6zD/08Sju864rkxFSd9UFTnP3b8p9lxgnlxriTzyw35Q3Lzv3c1/RcOT31lyfyvOj31i2myr6cg2TemIH/WVCuBv5D0l9tB3nwuO0u/PIz15cr8H5yrxF5bQv//RUPo964hzO+qITT2Qqxf/u3Et2oIzbzUEOylsv1oDcH+2zXk23ZkvaI871lF3tuWLPJl2vVbNYR6Odv1cqHPZ/QDyFkwP6vWwgrnL6g0zXzM8A3KfY4vksK/VB9c3HPwRmUFxae/9H5fvQvafAViD1jj/wA=</diagram></mxfile>
```

Comment:
Adding drawio diagram file to make it easier to update the db diagram. Include instructions in README